index.html

```diff
@@ -82,194 +115,210 @@ <h2>
     </section>
     <section>
       <h2>
-        Registry table of powerful features
+        Registry table of standardized permissions
       </h2>
       <p>
-        [=Powerful features=] are identified by a unique string token, which is used for
-        integration with the [[[Permissions]]] and [[[Permissions-Policy]]] specifications.
+        For a permission to appear in the table of standardized permissions, and thus be considered
+        a <dfn>standardized permission</dfn>, it needs to meet the following criteria:
+      </p>
+      <ul>
+        <li>Implemented and demonstrably interoperable in at least two browser engines (e.g., has
+        accompanying <a href="https://web-platform-tests.org">Web Platform Tests</a>).
+        </li>
+        <li>Is specified in published as [=technical report=] by a W3C [=Working Group=] with
+        maturity of [=first public working draft=] or above, or is published by the <a href=
+        "https://whatwg.org">WHATWG</a> as a standard.
+        </li>
+      </ul>
+      <p>
+        Each [=permissions=] is identified by a unique <dfn data-dfn-for="permission">string
+        token</dfn>, which, in case of [=policy-controlled features=] it allows integration with
```

Comment:
maybe: "which, in the case of $FOO it allows for"? (you do the in the second half of the sentence, and I think it reads better)